tenable/io/v3/vm/networks.py

```diff
@@ -0,0 +1,329 @@
+'''
+Networks
+========
+
+The following methods allow for interaction into the Tenable.io
+:devportal:`networks <networks>` API endpoints.
+
+Methods available on ``tio.v3.vm.networks``:
+
+.. rst-class:: hide-signature
+.. autoclass:: NetworksAPI
+    :members:
+'''
+from typing import Dict, List, Union
+
+from tenable.errors import UnexpectedValueError
+from tenable.io.base import TIOEndpoint, TIOIterator
+
+
+class NetworksIterator(TIOIterator):
+    '''
+    The networks iterator provides a scalable way to work through networks
+    result sets of any size.  The iterator will walk through each page
+    of data, returning one record at a time.  If it reaches the end of a
+    page of records, then it will request the next page of information and
+    then continue to return records from the next page (and the next, and the
+    next) until the counter reaches the total number of records that the API
+    has reported.
+
+    Attributes:
+        count (int): The current number of records that have been returned
+        page (list):
+            The current page of data being walked through.  pages will be
+            cycled through as the iterator requests more information from the
+            API.
+        page_count (int): The number of record returned from the current page.
+        total (int):
+            The total number of records that exist for the current request.
+    '''
+    pass
+
+
+class NetworksAPI(TIOEndpoint):
+    '''
+    This will contain all methods related to networks
+    '''
+    _path = 'api/v3/networks'
+    _conv_json = True
+
+    def create(self,
+               name: str,
+               description: str = None,
+               assets_ttl_days: int = None
+               ) -> Dict:
+        '''
+        Creates a new network within Tenable.io
+
+        :devportal:`networks: create <networks-create>`
+
+        Args:
+            name: The name of the new network.
+            description (optional): Description of the network.
+            assets_ttl_days (optional):
+                The number of days to wait before assets age out.
+                Assets will be permanently deleted if they are not seen on
+                a scan within the specified number of days.
+                Minimum value: 90
+                Maximum value: 365
+
+        Returns:
+            :obj:`dict`:
+                The resource record of the newly created network.
+
+        Examples:
+            >>> nw = tio.v3.vm.networks.create('Example')
+        '''
+        if not description:
+            description = ''
+
+        return self._post('', json={
+            'name': name,
+            'description': description,
+            'assets_ttl_days': assets_ttl_days
+        })
+
+    def delete(self, network_id: str) -> None:
```

Comment:
we can use UUID as type 
```suggestion
    def delete(self, network_id: UUID) -> None:
```

Reply:
Updated at all instances